tools/bin/commands/install

```diff
@@ -130,6 +132,7 @@ install::run() {
 
     wait_for_deployments 1 syndesis-operator
 
+    local app_options=$(readopt --app-options)
```

Comment:
@zregvart 
Sorry, coming to this late. The intention was not to expose app-options as a user switch.
The operator hub has no parameter mechanism other than the custom resource hence the --custom-resource mirrors that use-case.
The app-options var is merely an internal variable to facilitate adding --custom-resource to the existing install command.
To address the 'unbound variable' problem, L135 should probably just be `local app-options=""`
WDYT?

Reply:
As long as the intent that @claudio4j has done here is maintained I'm fine with that. Should be as easy as this to enable addons.

Reply:
Let the user add parameters to `install app` is easier than changing the custom resource isn't ?  I would like to have this option to install addons.